<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deb</class>
 <widget class="QWidget" name="Deb">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 淡棕色温暖背景 - QWidget主窗口 */
QWidget {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #f5e6d3, stop:0.5 #f0d9c0, stop:1 #ebccad);
    border: none;
}

/* 分组框样式 - 白色卡片 */
QGroupBox {
    background-color: rgba(255, 255, 255, 0.9);
    border: 1px solid #d4b996;
    border-radius: 8px;
    margin-top: 10px;
    padding-top: 10px;
    color: #5c3713;
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 8px;
    background-color: #ffffff;
    color: #8b5a2b;
    border-radius: 4px;
}

/* Frame样式 */
QFrame {
    background-color: rgba(255, 255, 255, 0.85);
    border: 1px solid #d4b996;
    border-radius: 6px;
}

/* 普通标签样式 */
QLabel {
    background-color: transparent;
    color: #5c3713;
    font-size: 14px;
}

/* 特殊标签 - 时间显示等 */
QLabel#time_1 {
    color: #8b4513;
    font-size: 16px;
    font-weight: bold;
}

/* 状态标签 */
QLabel#labelDUANKAI,
QLabel#label_6wIFI {
    color: #d2691e;
    font-weight: bold;
}

/* 输入框样式 - 淡粉色 */
QLineEdit {
    background-color: #FFE4E6;  /* 淡粉色 */
    border: 1px solid #f8c6cb;
    border-radius: 4px;
    padding: 6px 8px;
    color: #5c3713;  /* 棕色文字 */
    font-size: 14px;
    selection-background-color: #f8c6cb;
}

QLineEdit:focus {
    border: 1px solid #f4a7b0;
    background-color: #ffdde0;
}

/* 纯文本编辑框 - 网易云黑 */
QPlainTextEdit {
    background-color: #2d2d2d;  /* 网易云黑 */
    border: 1px solid #404040;
    border-radius: 4px;
    padding: 6px 8px;
    color: #e0e0e0;  /* 浅灰色文字 */
    font-size: 14px;
    selection-background-color: #1db954;
}

/* 表格视图 - 网易云黑 */
QTableView {
    background-color: #2d2d2d;  /* 网易云黑 */
    border: 1px solid #404040;
    border-radius: 4px;
    color: #e0e0e0;
    gridline-color: #404040;
    alternate-background-color: #333333;
}

QTableView::item {
    background-color: #2d2d2d;
    color: #e0e0e0;
    border: none;
    padding: 4px;
}

QTableView::item:selected {
    background-color: #1db954;
    color: #ffffff;
}

QTableView::item:hover {
    background-color: #3a3a3a;
}

QHeaderView::section {
    background-color: #404040;
    color: #e0e0e0;
    padding: 6px;
    border: none;
    font-weight: bold;
}

/* 复选框样式 */
QCheckBox {
    background-color: transparent;
    color: #5c3713;
    spacing: 8px;
}

QCheckBox::indicator {
    width: 16px;
    height: 16px;
    border: 1px solid #b08f6d;
    border-radius: 3px;
    background-color: #ffffff;
}

QCheckBox::indicator:checked {
    background-color: #d2691e;
    border: 1px solid #d2691e;
}

/* 滑动条样式 */
QSlider::groove:horizontal {
    border: 1px solid #b08f6d;
    height: 4px;
    background-color: #e8d5c4;
    border-radius: 2px;
}

QSlider::handle:horizontal {
    background-color: #d2691e;
    border: 1px solid #d2691e;
    width: 16px;
    height: 16px;
    border-radius: 8px;
    margin: -6px 0;
}

QSlider::handle:horizontal:hover {
    background-color: #e37c2b;
    border: 1px solid #e37c2b;
}

/* 按钮样式 - 保持原有粉色效果 */
QPushButton,
QToolButton {
    border: none;
    background-color: transparent;
    outline: none;
    padding: 8px 12px;
    border-radius: 4px;
    font-size: 14px;
    color: #5c3713;  /* 棕色文字 */
}

QPushButton:hover,
QToolButton:hover {
    background-color: #FFE4E6;  /* 淡粉色 */
    border: none;
    color: #333333;
}

QPushButton:pressed,
QToolButton:pressed {
    background-color: #F3E8FF;  /* 淡紫色 */
    border: none;
    color: #333333;
}

QPushButton:focus,
QToolButton:focus {
    outline: none;
    border: none;
}

/* 特殊按钮样式 */
QPushButton#clear_yu_bt,
QPushButton#set_yu_bt,
QPushButton#set_light_bt {
    background-color: #d4b996;
    color: #5c3713;
}

QPushButton#clear_yu_bt:hover,
QPushButton#set_yu_bt:hover,
QPushButton#set_light_bt:hover {
    background-color: #e0c9a8;
}

/* 线条分隔符 */
Line {
    background-color: #d4b996;
    border: none;
}

/* 进度条样式 */
QProgressBar {
    border: 1px solid #b08f6d;
    border-radius: 4px;
    text-align: center;
    color: #5c3713;
    background-color: #e8d5c4;
}

QProgressBar::chunk {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                      stop:0 #d2691e, stop:1 #e37c2b);
    border-radius: 3px;
}</string>
  </property>
  <widget class="QPushButton" name="exit_bt">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>36</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/exit.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="send_edi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>316</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/clearS.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="rec_edi">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>441</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_bt">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>266</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/clear_b.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>调  试  界  面</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="send_bt">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>360</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/send.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>99</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>定时器</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>410</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 170, 255);</string>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>408</y>
     <width>69</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>mms</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
